lib/components/user/monitored-trip/trip-status-rendering-strategies/base-renderer.js

```diff
@@ -1,18 +1,21 @@
 import moment from 'moment'
-import { formatDuration } from '@opentripplanner/core-utils/lib/time'
+import { useIntl } from 'react-intl'
+
+import { FormattedDuration } from '../../../narrative/default/format-duration'
 
 /**
  * Calculate commonly-used pieces of data used to render the trip status
  * component. The monitoredTrip param can be undefined.
  */
 export default function baseRenderer (monitoredTrip) {
+  const intl = useIntl()
```

Comment:
Tricky: `baseRenderer` is not a component and not a functional component, so this call fails when rendering a trip detail page.

Reply:
This would hold for changes to active-trip-renderer as well

Reply:
Both should be addressed in https://github.com/opentripplanner/otp-react-redux/pull/408/commits/6c54a1e02879df30d2805cd8bc3d09152b79d7d7